tests/dflydev/tests/util/antPathMatcher/AntPathMatcherTest.php

```diff
@@ -48,11 +48,16 @@ public function formatTestMatchesMessage($pattern, $path)
         return 'Testing path "' . $path . '" against pattern "' . $pattern . '"';
     }
 
-    public function testMatchesProvider()
+    public function provideMatches()
     {
         return array(
             array(
                 'com/t?st',
+                array('com/test', 'com/tast', 'com/txst'),
+                array('com/test.jsp', 'com/tast.jsp', 'com/txst.jsp'),
+            ),
```

Comment:
Another addition: I kept the original `com/t?st` pattern and changed the expected matches and non-matches. This should make it more clear that there's no special behavior to allow an optional file extension (at least not without a trailing `*` in the pattern).